cheroot/server.py

```diff
@@ -469,14 +523,14 @@ def _fetch(self):
         except ValueError:
             raise ValueError(
                 'Bad chunked transfer size: {chunk_size!r}'.
-                format(chunk_size=chunk_size),
-            )
+                    format(chunk_size=chunk_size),
```

Comment:
Please don't change the formatting here. It was good originally, just like I intended — with closing bracket matching the initial block indentation.

Reply:
Sorry about that, the IDE I'm using likes to be a bit overzealous and auto-format entire files, I'll pull out these changes in an upcoming commit.

Reply:
Thanks. I just wanted to make some early review and ended up checking each diff chunk for whether it's some formatting change or is it actually useful...